plugins/device/deviceutils/store-utils.go

```diff
@@ -174,3 +178,77 @@ func IsVgExist(vgname string) bool {
 	}
 	return false
 }
+
+// getDeviceAvailableSize gets the device size and vgName using device Path
+func getDeviceAvailableSize(devicePath string) (string, uint64, error) {
+	devices, err := GetDevices()
+	if err != nil {
+		return "", 0, err
+	}
+	deviceVgName := strings.Split(devicePath, "/")[2]
+	for _, d := range devices {
+		if d.VgName == deviceVgName {
+			return d.VgName, d.AvailableSize, nil
+		}
+	}
+	return "", 0, nil
```

Comment:
don't we need to return an error if the `devicePath`  is not found?

Reply:
Done